src/content/reference/react/createRef.md

```diff
@@ -107,17 +107,16 @@ export default class Form extends Component {
 
 <Pitfall>
 
-`createRef` is mostly used for [class components.](/reference/react/Component) Function components typically rely on [`useRef`](/reference/react/useRef) instead.
-
+`createRef` kebanyakan digunakan untuk [komponen kelas.](/reference/react/Component) sedangkan komponen fungsional biasanya mengandalkan [`useRef`](/reference/react/useRef).
 </Pitfall>
```

Comment:
```suggestion
`createRef` kebanyakan digunakan untuk [komponen kelas.](/reference/react/Component) sedangkan komponen fungsional biasanya mengandalkan [`useRef`](/reference/react/useRef).

</Pitfall>
```

Please don't add/remove any empty lines. This will cause merge conflicts when we merge from upstream.